packages/api/cms-api/src/dam/files/dto/file.input.ts

```diff
@@ -89,6 +89,12 @@ export class CreateFileInput {
     // TODO is this validation even used?
     @IsObject()
     scope?: DamScopeInterface;
+
+    @IsOptional()
```

Comment:
Please use the new `@IsUndefinable()` and `@IsNullable()` decorators according to the use case.